packages/components/src/VirtualizedList/RowLarge/RowLarge.component.js

```diff
@@ -13,7 +13,7 @@ import theme from './RowLarge.scss';
 /**
  * Row renderer that displays a Large item
  */
-function RowLarge({ className, index, key, parent, style }) {
+function RowLarge({ className, index, key, parent, style, onRowClick }) {
```

Comment:
Remove that and get it from parent. It should be something like `parent.props.onRowClick`